dev/docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.7.0
+ENV PYTHONUNBUFFERED 1
+
+# Update
+RUN apt update -y && apt upgrade -y && apt install gcc libffi-dev wget git -y
+
+EXPOSE 11500
+
+RUN mkdir /code
+WORKDIR /code
+ADD . /code
+
+# install reqs and solc
+RUN pip install --upgrade pip \
+    && pip install pipenv \
```

Comment:
Perhaps skip pipenv all together, and use *only* pip.

Reply:
I had that in there because the install_solc script uses it.  I could see if it works without it.

Reply:
Ah - gotcha. That's a cheap pipenv trick anyways.  You may be able to call the install script directly.